packages/web/src/shared/components/RichText.tsx

```diff
@@ -22,7 +22,7 @@ export default function RichText({ content, className = '' }: RichTextProps) {
     editor?.commands.setContent(content, false, {
       preserveWhitespace: 'full',
     });
-  }, [editor?.commands, content]);
```

Comment:
This line needs to stay the same. It ensures this effect is only executed if the commands object changes

Reply:
Done.